src/utils/renderDate/index.js

```diff
@@ -0,0 +1,6 @@
+export default function renderDate(dateString) {
+	const date = new Date(dateString);
+	const [month, day, year] = [date.getMonth(), date.getDate(), date.getFullYear()];
+	const formattedDate = `${day}.${month + 1}.${year}`;
+	return formattedDate;
```

Comment:
To simply display the date in a specific format, you could also use the method `.toLocaleDateString()` with the option `'de-De'` - check it out: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString

Reply:
Awesome, thank you! I actually tried .toLocaleString() but thought it's useless outside of America. I missed the "de-DE" option.